client/lib/post-normalizer/utils.js

```diff
@@ -248,6 +249,22 @@ export function deduceImageWidthAndHeight( image ) {
 			height,
 		};
 	}
+
+	let parsed;
+	try {
+		parsed = url.parse( image.src );
+	} catch ( e ) {
+		return null;
+	}
+
+	if ( isPhotonHost( parsed.host ) ) {
+		const queryArgs = qs.parse( parsed.query );
+		return {
+			width: queryArgs.w,
```

Comment:
what happens down the chain if we have a valid `w` or `h` but not the other?

Reply:
That's a good question.  For the purposes of these previews, the determination is based on the sum of the height & width.  

I'll dig a bit to see if one or the other being missing will have adverse / unexpected effects.